### DISCUSSION TITLE
How to restore files from `rm --trash`?

### DISCUSSION THREAD

**Discussion by 2639-unofficial** [nushell/nushell#10190] [Q&A] [answered]
Hi, I recently came across [oberblastmeister/trashy](https://github.com/oberblastmeister/trashy), which could restore files from the trash bin, but would require custom completions.

Is there a native way for restoring recycled files?

**Comment by fdncred** [comment, marked as answer]
Restoring files is typically a process for the operating system. I don't believe there is any nushell built-in to handle restoring files from the trash.

**Reply by 2639-unofficial** [reply]
Thanks for answering. Custom completion for trashy is the way to go then.
